spec/rubocop/cop/lint/redundant_cop_disable_directive_spec.rb

```diff
@@ -503,5 +503,92 @@ def bar
         end
       end
     end
+
+    context 'with all department disabling' do
```

Comment:
"all department disabling" sounds wrong to me. Probably you meant to write "with a disabled department" or something like this.

Reply:
Replaced